exercises/introducao-a-javascript-e-logica-de-programacao_1/dia_1/exercicio_4.js

```diff
@@ -0,0 +1,8 @@
+let a = -10;
+
+if (a >= 0){
+    console.log("Número positivo");
+}
+else {
+    console.log("Número negativo");
+}
```

Comment:
Quando o número é zer o console retorna positivo! Pela especificação do exercício, tem que ter uma mensagem específica quando o número for zero:

> Faça um programa que, dado um valor definido numa variável, retorne "positive" se esse valor for positivo, "negative" se for negativo e "zero" caso contrário.